README.md

```diff
@@ -65,7 +65,7 @@ pip install ipyradiant
 Install the lab extensions that `ipyradiant` depends on.
 
 ```bash
-jupyter labextension install @jupyter-widgets/jupyterlab-manager jupyter-cytoscape @pyviz/jupyterlab_pyviz qgrid2
+jupyter labextension install @jupyter-widgets/jupyterlab-manager jupyter-cytoscape @pyviz/jupyterlab_pyviz
```

Comment:
for lab 3, you won't do any of this: they're already part of the conda/pip packages

Reply:
1e6f88a